### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by ramirolc02**
yarn run ts-node --files ./scripts/checkVotePower.ts "tokenContract" "addressToCheckVotePower"

### CODE REVIEW

**Review by fernandoandreotti — APPROVED** [review]
nice

File: week_3/scripts/checkVotePower.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { ethers } from "hardhat";
+import * as dotenv from 'dotenv';
+import { Ballot__factory, MyToken__factory } from "../typechain-types";
```
Comment (fernandoandreotti) [inline week_3/scripts/checkVotePower.ts:3]: Ballot__factory is unused

File: week_3/scripts/checkVotePower.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    const wallet = new ethers.Wallet(privateKey);
+    const signer = wallet.connect(provider);
+    const contract = new MyToken__factory(signer);
+    const deployedContract = await contract.attach(tokenContractAddress);
+    console.log(`Attached to contract: ${tokenContractAddress}`);
+
+    const getVotePower = await deployedContract.getVotes(checkVotePowerAddress);
+    console.log(`Account ${checkVotePowerAddress} has ${getVotePower} vote power units`);
```
Comment (fernandoandreotti) [inline week_3/scripts/checkVotePower.ts:28]: ${ethers.utils.formatEther(getVotePower)} maybe? to get from 3000000000 to 300?

Comment (ramirolc02) [reply week_3/scripts/checkVotePower.ts:28]: Yes, will look better.

**Comment by fernandoandreotti** [comment]
Worked for me

**Pull request merged**